src/ontology/edits/oeo-physical.omn

```diff
@@ -369,6 +481,48 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/234"
         PortionOfMatter
     
     
+Class: <http://openenergy-platform.org/ontology/oeo-physical/HardCoalPowerUnit>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A HardCoalPowerUnit is a CoalPowerUnit using HardCoal as Fuel."
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo-physical/CoalPowerUnit>,
+        uses some HardCoal
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo-physical/HydroPowerUnit>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A HydroPowerUnit is a PowerGeneratingUnit using hydrogen as fuel.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A HydroPowerUnit is a PowerGeneratingUnit using the potential and/or kinetic energy of water.",
```

HydroPowerUnit and Hydrogen are false friends. Next to the HydroPowerUnit we also need a HydrogenFiredPowerUnit.